hack/netpol-generator/install-cyclonus.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
```

Comment:
maybe the documentation above should mention that you can consume and run cyclonus as a K8 job?

Reply:
👍